src/game/isGameOver.js

```diff
@@ -0,0 +1,5 @@
+import { getBoardStatus } from "../board/getBoardStatus";
+
+export function isGameOver() {
+  return getBoardStatus() !== "Ongoing";
+}
```

Comment:
This method feels a bit funky to me. There should probably be a test around this. 

As a standalone function, how would isGameOver know what `getBoardStatus` should be? If you tested this, you might find that getBoardStatus actually expects an argument ` export function getBoardStatus(board) `

--

Actually, after looking at your code again, I don't think `isGameOver` is used anywhere? Should this file actually not be here?

Reply:
Oh! You're right! That was some legacy code from the spike. Thanks for catching that. I'll be more conscientious of this in the future.